projects/demo/src/modules/components/link/examples/3/index.html

```diff
@@ -12,7 +12,8 @@
         iconEnd="@tui.circle-help"
         tuiLink
         type="button"
-        [pseudo]="true"
+        [style.text-decoration-line]="'underline'"
```

Comment:
This line should be unnecessary as line would be hidden only in default `action` appearance.

Reply:
<img width="342" alt="image" src="https://github.com/user-attachments/assets/a74c0f8a-13b7-43af-a749-c055f4067cdd" />

 Without `[style.text-decoration-line]="'underline'"` this is an angular 16 bug

Reply:
fixed